FLAIR.md

```diff
@@ -0,0 +1,29 @@
+Acknowledgments
+=========
+
+<p valign="middle"><a href="https://www.browserstack.com/"><img src="http://media.archiveofourown.org/ao3/logos/Browserstack.svg" width="200"/></a> for cross-browser testing tool. </p>
```

Comment:
For some of these (e.g. Capybara), you're writing out the name in the text. For others, you're letting the image stand in for text. I'd suggest writing out the name no matter what, since it means the name will be available even if the image breaks or is hard to read for some people. 

In addition to that, the images are all links, so they need alt text: http://webaim.org/techniques/alttext/ & http://webaim.org/blog/alt-text-and-linked-images/